### PR TITLE
Add request timeout

### PR DESCRIPTION
**Pull request by Bolodya1997**
Adds `NSM_REQUEST_TIMEOUT`. Closes https://github.com/networkservicemesh/cmd-nsc/issues/91.

### CODE REVIEW

File: internal/config/config.go [hunk 1]
```diff
@@ -38,6 +38,7 @@ type Config struct {
 	Name             string        `default:"nsc" desc:"Name of Network Service Client"`
 	ConnectTo        url.URL       `default:"unix:///var/lib/networkservicemesh/nsm.io.sock" desc:"url to connect to NSM" split_words:"true"`
 	ConnectTimeout   time.Duration `default:"5s" desc:"timeout to connect to NSMgr" split_words:"true"`
+	RequestTimeout   time.Duration `default:"5m" desc:"timeout to request NSE" split_words:"true"`
```
Comment (denis-tingaikin) [inline internal/config/config.go:41]: Why 5 min?

**Review by denis-tingaikin — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -10,7 +10,8 @@ This README will provide directions for building, testing, and debugging that co
 
 * NSM_NAME - A string value of network service client name (default "nsc")
 * NSM_CONNECT_TO - A Network service Manager connectTo URL (default "unix:///var/lib/networkservicemesh/nsm.io.sock")
-* NSM_CONNECT_TIMEOUT - A timeout to connect to Network service Manager (default 5s)
+* NSM_CONNECT_TIMEOUT - A timeout to connect to Network Service Manager (default 5s)
```
Comment (denis-tingaikin) [inline README.md:13]:
```suggestion
* NSM_DIAL_TIMEOUT - A timeout to connect to Network Service Manager (default 5s)
```

File: internal/config/config.go (hunk 1 above)
Comment (Bolodya1997) [reply internal/config/config.go:41]: 5 min is not too long and not too short :)

Comment (Bolodya1997) [reply internal/config/config.go]: Reduced to 15s.

File: README.md (hunk 2 above)
Comment (Bolodya1997) [reply README.md]: Renamed.

File: internal/config/config.go (hunk 1 above)
Comment (denis-tingaikin) [reply internal/config/config.go]: It is long

**Review by denis-tingaikin — APPROVED** [review]

**Pull request merged**